specification/marketplacecatalog/Reviews.Management/reviews.models.tsp

```diff
@@ -30,6 +32,30 @@ model UserHasReviewProperties {
   updatedAt: string;
 }
 
+@doc("Review email notification request.")
+@added(Versions.v2025_05_01)
+model ReviewEmail is ProxyResource<ReviewEmailProperties> {
+  ...ResourceNameParameter<
+    Resource = ReviewEmail,
+    KeyName = "uniqueProductId",
+    SegmentName = "products"
+  >;
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "Read-only metadata resource."
+@doc("Properties of the review email notification request.")
+@added(Versions.v2025_05_01)
+model ReviewEmailProperties {
+  @doc("The user ID.")
+  userId: string;
+
+  @doc("The full name of the user.")
+  fullName: string;
+
+  @doc("The email address of the user.")
+  email: string;
```

Comment:
curious, did you get a privacy review done to accept these user details into your API ?